runtime/src/main/java/io/quarkiverse/cxf/CxfConfig.java

```diff
@@ -15,4 +18,18 @@
      */
     @ConfigItem(name = "endpoint")
     public Map<String, CxfEndpointConfig> endpoints;
+
+    /**
+     * Configure client proxiess.
+     */
+    @ConfigItem(name = "client")
+    public Map<String, CxfClientConfig> clients;
+
+    public boolean isClientPresent(String key) {
+        return Optional.ofNullable(clients).map(m -> m.containsKey(key)).orElse(false);
+    }
+
+    public @Nullable CxfClientConfig getClient(String key) {
```

Comment:
Please remove Nullable as per previous comment.

Reply:
Removed, however ..

I really like the following style in Java:
An argument can only be null if the parameter is annotated with something like `@Nullable`. A `@Nullable` parameter is an exception that should be only used when necessary. I really would like to see `@Nullable`  as Java standard.

Reply:
Done.